_posts/2018-11-22-NUnitTestGenerator-2.1.md

```diff
@@ -0,0 +1,8 @@
+NUnit Test Generator extension for Visual Studio 2017, version 2.1 is released.

```

Comment:
The comma seems to split the sentence in an awkward way. Don't think I've seen a comma separate the product name and the version while inside a sentence.